host/xtest/Makefile

```diff
@@ -40,7 +40,7 @@ endef
 WITH_OPENSSL ?= y
 ifeq ($(WITH_OPENSSL),y)
 CFLAGS += -DOPENSSL_FOUND=1 -DOPENSSL_API_COMPAT=10100
-LDFLAGS += -lcrypto
+LDFLAGS += -L$(OPENSSL_DIR)/lib -lcrypto
```

Comment:
Is `OPENSSL_DIR` a standard make or environment variable? I seems to me it's awaited only for Rust based builds. Won't this change break existing build environments that use this Makefile but (maybe) do not define `OPENSSL_DIR`?
Maybe use:
```patch
 ifeq ($(WITH_OPENSSL),y)
 CFLAGS += -DOPENSSL_FOUND=1 -DOPENSSL_API_COMPAT=10100
 LDFLAGS += -lcrypto
+ifneq (,$(OPENSSL_DIR))
+LDFLAGS += -L$(OPENSSL_DIR)/lib
+endif

Reply:
I have now changed the variable to `OPENSSL_LIBRARY_DIR` to avoid any confusion with the standard variables.